.travis.yml

```diff
@@ -45,7 +43,6 @@ matrix:
       gemfile: gemfiles/Gemfile.rails-master
       env: DEVISE_ORM=mongoid
```

Comment:
I see you added ruby 2.6 and then removed, it could be added to test only rails 5.0 and 5.2, adding these lines here:

```yaml
    - rvm: 2.6.0
      gemfile: gemfiles/Gemfile.rails-5.0
      env: DEVISE_ORM=active_record
    - rvm: 2.6.0
      gemfile: gemfiles/Gemfile.rails-5.0
      env: DEVISE_ORM=mongoid
    - rvm: 2.6.0
      gemfile: gemfiles/Gemfile.rails-5.2
      env: DEVISE_ORM=active_record
    - rvm: 2.6.0
      gemfile: gemfiles/Gemfile.rails-5.2
      env: DEVISE_ORM=mongoid
```